_clients/opensearch-py-ml.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: default
+title: Opensearch-py-ml
+nav_order: 11
+---
+
+`opensearch-py-ml` is a Python client that provides a suite of data analytics and natural language processing (NLP) support tools for OpenSearch. It provides data analysts with the ability to:
+
+- Call OpenSearch indexes and manipulate using the opensearch-py-ml [DataFrame](https://opensearch-project.github.io/opensearch-py-ml/reference/dataframe.html) APIs. The opensearch-py-ml DataFrame wraps an OpenSearch index into a API similar to [pandas](https://pandas.pydata.org/), giving you the ability to process large amounts of data from OpenSearch inside a Jupyter Notebook.
+-  Upload NLP [SentenceTransformers](https://www.sbert.net/) models into OpenSearch using the [ML Commons plugin]({{site.url}}{{site.baseurl}}/ml-commons-plugin/index/).
```

Comment:
Should it be "SentenceTransformer" (no "s")?

Reply:
https://github.com/UKPLab/sentence-transformers/blob/master/sentence_transformers/SentenceTransformer.py referring to `sentence-transformers` github,  `sentence-transformers` stands for the type of NLP models.

-- if we are using plural form for models, we should call SentenceTransformers Models. 
-- when using singular form then it's sentenceTransformer mode.

Reply:
> https://github.com/UKPLab/sentence-transformers/blob/master/sentence_transformers/SentenceTransformer.py referring to `sentence-transformers` github, `sentence-transformers` stands for the type of NLP models.
> 
> -- if we are using plural form for models, we should call SentenceTransformers models. -- when using singular form then it's sentenceTransformer model.

This doesn't sound accurate to me. If the singular is SentenceTransformer model, then the plural would be SentenceTransformer models.